src/foam/nanos/u2/navigation/FooterView.js

```diff
@@ -137,7 +137,7 @@ foam.CLASS({
       name: 'goToTerm',
       label: '',
       code: function(X) {
-        X.openTermsModal();
+        this.window.location.assign(X.appConfig.termsAndCondLink);
```

Comment:
Better to do X.window.location because it allows the application to work with multiple windows.

Reply:
Fixed. https://github.com/foam-framework/foam2/pull/2649/commits/4a2bd6b91d456f279ec30f19f193ba9f3adb4e4d